catalog/dags/maintenance/add_license_url.py

```diff
@@ -227,12 +186,19 @@ def add_license_url():
         GROUP BY license, license_version
     """)
 
-    license_groups = get_license_groups(query)
-    updated = update_license_url.partial(batch_size="{{ params.batch_size }}").expand(
-        license_group=license_groups
-    )
-    report_completion(updated, query)
-    updated >> report_failed_license_pairs()
+    licenses = get_licenses(query)
```

Comment:
Nit: I think it would be easier to understand the code flow if the query is moved inside `get_licenses` function.